Patches/AnimalCollabProj/AnimalCollabProj_CE_Patch_Races_Animal_Goose.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>AnimalCollabProj</modName>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[@Name="GooseThingBase"]/statBases</xpath>
+				<value>
```

Comment:
Goose was also added as a farm animal, just copy the stats.

Reply:
Ah, I see.

Fixed in ed9e97d